src/dm-screen/components/DmView/DmView.tsx

```diff
@@ -127,10 +221,17 @@ export const DmView = () => {
       };
     });
 
-    setItems([
-      ...items,
-      ...newItems
-    ])
+    const initiativeOrder = getInitiativeOrder();
+
+    if (initiativeOrder) {
+      initiativeOrder.setItems([
+        ...initiativeOrder?.getItems() ?? [],
```

Comment:
I wonder if getItems should return [] if there are none, instead of making a check here?

Reply:
It probably should. Better null checking is on the list of things to do for this ticket.